fairlearn/postprocessing/_constants.py

```diff
@@ -9,8 +9,6 @@
 
 OUTPUT_SEPARATOR = "-"*65
 
-_MATPLOTLIB_IMPORT_ERROR_MESSAGE = "Please make sure to install fairlearn[customplots] to use " \
```

Comment:
Why is this being removed?

Reply:
I think because he's added matplotlib to the package as a "main" dependency, so this case doesn't happen anymore. The main question is whether we want that actually.

Reply:
we could go either way. We could still have `matplotlib` as a soft dependency which is only needed when people use the plotting functionality, or have it as a core dependency. I would lean towards having it as a soft dependency.

Reply:
Looks like @riedgar-ms, me, @MiroDudik (asked him separately), and @adrinjalali are all leaning towards soft dependency (i.e. status quo).